bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_tch_4f_ckm_13TeV_UL/st_tch_4f_ckm_width_variations/create_gridpacks.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env zsh
+
+# Run from bin/MadGraph5_aMCatNLO/ as ./cards/production/2017/13TeV/st_tch_4f_ckm_13TeV_UL/st_tch_4f_ckm_width_variations/create_gridpacks.sh
+
+git apply cards/production/2017/13TeV/st_tch_4f_ckm_13TeV_UL/st_tch_4f_ckm_width_variations/gridpack_generation_topwidth.patch
+
+NOMINAL_WIDTH=1.476400
```

Comment:
Question about the analysis strategy: Is 1.4764 the default width value in other relevant samples? Will this be a simultaneous fit to several processes?

Reply:
The nominal `ST_t-channel_[anti]top_4f_InclusiveDecays_TuneCP5_13TeV-powheg-madspin-pythia8` samples use the default MadSpin decays, hence use the LO width of 1.4764, so it makes more sense for the variations to be computed w.r.t. that one.
For a width measurement, as long as the actual values used in the samples are known, I don't see why that should be a problem.